collect_app/src/main/java/org/odk/collect/android/formmanagement/matchexactly/SyncStatusAppState.kt

```diff
@@ -4,14 +4,10 @@ import android.content.Context
 import androidx.lifecycle.LiveData
 import androidx.lifecycle.MutableLiveData
 import org.odk.collect.android.external.FormsContract
+import org.odk.collect.androidshared.data.AppState
 import org.odk.collect.forms.FormSourceException
-import javax.inject.Singleton
 
-@Singleton
-class SyncStatusAppState(private val context: Context) {
-
-    private val syncing = mutableMapOf<String, MutableLiveData<Boolean>>()
-    private val lastSyncFailure = mutableMapOf<String, MutableLiveData<FormSourceException?>>()
+class SyncStatusAppState(private val appState: AppState, private val context: Context) {
```

Comment:
We've been replacing our usages of `@Singleton` with `AppState`, so I did that here seeing as I was touching the class.